webui/react/src/pages/ExperimentDetails/Header/ExperimentHeaderFoldableContent.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import TagList from 'components/TagList';
+import useExperimentTags from 'hooks/useExperimentTags';
+import { ExperimentBase } from 'types';
+
+import css from './ExperimentHeaderFoldableContent.module.scss';
+
+interface Props {
+  experiment: ExperimentBase;
+  fetchExperiment: () => void;
+}
+
+const ExperimentHeaderFoldableContent: React.FC<Props> = (
```

Comment:
@hkang1 we might consider including this directly in `webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx`